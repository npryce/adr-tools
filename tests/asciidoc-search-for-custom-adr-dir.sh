adr init -t adoc architecture-log
adr new First Record
mkdir subdir
cd subdir
adr new Second Record
adr list
cd ..
adr list
