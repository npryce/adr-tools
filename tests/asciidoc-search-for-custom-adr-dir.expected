adr init -t adoc architecture-log
architecture-log/0001-record-architecture-decisions.adoc
adr new First Record
architecture-log/0002-first-record.adoc
mkdir subdir
cd subdir
adr new Second Record
../architecture-log/0003-second-record.adoc
adr list
../architecture-log/0001-record-architecture-decisions.adoc
../architecture-log/0002-first-record.adoc
../architecture-log/0003-second-record.adoc
cd ..
adr list
architecture-log/0001-record-architecture-decisions.adoc
architecture-log/0002-first-record.adoc
architecture-log/0003-second-record.adoc
